hangman/test/hangman_test.exs

```diff
@@ -0,0 +1,74 @@
+defmodule HangmanTest do
+  use ExUnit.Case
+  doctest Hangman
```

Comment:
# -3

It's really nice that you have the tests. However, I'm surprised that you have the repetition of the game struct setup in each.

One obvious thing would be to have it extracted into a helper function

~~~ elixir
game = setup_game()
~~~

But, even better would be to change the definition of `new_game()` to be:

~~~ elixir
  def new_game(word \\ Dictionary.random_word()) do
    secret  =  word |> String.codepoints
    %Hangman.Game{
      secret:     secret,
      word_chars: MapSet.new(secret),
      letters:    String.duplicate("_", length(secret)) |> String.codepoints
    }
  end
~~~

That way you'd same a bunch of code here, and also test the setup code in the Game module.